File: sdk/java-vertx-web/src/main/java/org/openapitools/vertxweb/server/api/ProductLookupApiImpl.java
package org.openapitools.vertxweb.server.api;

import org.openapitools.vertxweb.server.model.InvalidCode;
import org.openapitools.vertxweb.server.model.Item;
import org.openapitools.vertxweb.server.model.NotFound;

import org.openapitools.vertxweb.server.ApiResponse;
import org.openapitools.vertxweb.server.ApiException;

import io.reactivex.Single;

import java.util.List;
import java.util.Map;

// Implement this class

public class ProductLookupApiImpl implements ProductLookupApi {
    public Single<ApiResponse<Item>> getItemByASIN(String asin) {
        return Single.error(new ApiException("Not Implemented").setStatusCode(501));
    }

    public Single<ApiResponse<Item>> getItemByEAN(String ean) {
        return Single.error(new ApiException("Not Implemented").setStatusCode(501));
    }

    public Single<ApiResponse<Item>> getItemByUPC(String upc) {
        return Single.error(new ApiException("Not Implemented").setStatusCode(501));
    }

}

File: sdk/scala-play-server/app/api/ProductLookupApiImpl.scala
package api

import model.InvalidCode
import model.Item
import model.NotFound

/**
  * Provides a default implementation for [[ProductLookupApi]].
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-03-27T18:11:16.680-04:00[America/New_York]")
class ProductLookupApiImpl extends ProductLookupApi {
  /**
    * @inheritdoc
    */
  override def getItemByASIN(asin: String): Item = {
    // TODO: Implement better logic

    Item(None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None)
  }

  /**
    * @inheritdoc
    */
  override def getItemByEAN(ean: String): Item = {
    // TODO: Implement better logic

    Item(None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None)
  }

  /**
    * @inheritdoc
    */
  override def getItemByUPC(upc: String): Item = {
    // TODO: Implement better logic

    Item(None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None, None)
  }
}
